crates/cli-support/src/js/mod.rs

```diff
@@ -2194,6 +2194,29 @@ impl<'a> Context<'a> {
         true
     }
 
+    fn expose_get_from_externref_table(
+        &mut self,
+        table: TableId,
+        alloc: FunctionId,
+    ) -> Result<MemView, Error> {
+        let view = self.memview_table("getFromExternrefTable", table);
+        assert!(self.config.externref);
+        if !self.should_write_global(view.to_string()) {
+            return Ok(view);
+        }
+        let table = self.export_name_of(table);
+        self.global(&format!(
+            "
+                function {}(idx) {{
+                    return wasm.{}.get(idx);
+                }}
+            ",
+            view, table,
+        ));
+
+        Ok(view)
+    }
```

Comment:
I think it'd be fine to avoid an extra intrinsic function for this, I think it could just inline the `wasm.table.get(idx)` at each call-site using `self.export_name_of(table)`

Reply:
We did end up using it for calling `drop` as well.